src/Apps/Artist/Routes/Overview/ArtistOverviewRoute.tsx

```diff
@@ -45,6 +45,8 @@ const ArtistOverviewRoute: React.FC<ArtistOverviewRouteProps> = ({
       <Meta name="title" content={title} />
       <Meta name="description" content={description} />
 
+      <Spacer y={[2, 0]} />
```

Comment:
This seems the parent component, right @dzucconi ? 😅

Reply:
Sort of; for our purposes let's say yes — but really `ArtistApp` would be. It's just kind of simpler to target it here rather than swap out different margins per-route in the actual parent. This is still an external margin (there's no UI above this in this component) but it's the lesser evil.

Reply:
🫠